lib/modules/manager/poetry/extract.spec.ts

```diff
@@ -141,8 +142,17 @@ describe('modules/manager/poetry/extract', () => {
     });
 
     it('extracts dependencies from dependency groups', async () => {
-      const content =
-        '[tool.poetry.dependencies]\ndep = "^2.0"\n\n[tool.poetry.group.dev.dependencies]\ndev_dep = "^3.0"\n\n[tool.poetry.group.typing.dependencies]\ntyping_dep = "^4.0"';
+      const content = `
```

Comment:
```suggestion
      const content = codeBlock`
```
then you can properly align code indented and the helper will remove leading whitespace

Reply:
Thanks viceice! I initially ran 
```
git grep codeblock
```
or asked google about it and did not find anything.
```
git grep codeBlock
```
gave me the examples so that I could be a very precise model of what is the current way of writing those.
(https://github.com/renovatebot/renovate/pull/21949/commits/353164341c5ffb09c4e17b7f838bc51559ec0029)